classes/rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly make sure that the `person` and `book` has `attr_reader` and `attr_writer`, you can add `rentals` to `attr_accessor` to fix this
```suggestion
  attr_accessor :date,  :person, :book
```